src/avfoundation.cs

```diff
@@ -2249,6 +2337,22 @@ interface AVAudioSession {
 		[Watch (6,0), TV (13,0), NoMac, iOS (13,0)]
 		[Export ("allowHapticsAndSystemSoundsDuringRecording")]
 		bool AllowHapticsAndSystemSoundsDuringRecording { get; }
+
+		[NoWatch, NoTV, NoMac, iOS (14, 0)]
+		[Export ("preferredInputOrientation")]
+		AVAudioStereoOrientation PreferredInputOrientation { get; }
+
+		[NoWatch, NoTV, NoMac, iOS (14,0)]
+		[Export ("setPreferredInputOrientation:error:")]
+		bool SetPreferredInputOrientation (AVAudioStereoOrientation orientation, [NullAllowed] out NSError outError);
+
+		[NoWatch, NoTV, NoMac, iOS (14, 0)]
+		[Export ("inputOrientation")]
+		AVAudioStereoOrientation InputOrientation { get; }
+
+		[NoWatch, NoTV, NoMac, iOS (14, 0)]
+		[Field ("AVAudioSessionPolarPatternStereo")]
+		NSString PolarPatternStereo { get; }
```

Comment:
There are other (older) members defined for `AVAudioSessionPolarPattern` and this one should go with them

https://developer.apple.com/documentation/avfoundation/avaudiosessionpolarpattern

Reply:
You mean move it to the same location in the file?

Reply:
it should be along other `AVAudioSessionPolarPattern*` fields which, sadly, are old enough that a smart enum was not used for them :(

Reply:
We wont forget now: https://github.com/xamarin/xamarin-macios/issues/9356 will done once xcode12 is done.